tests/infra/consortium.py

```diff
@@ -333,24 +333,35 @@ def retire_node(self, remote_node, node_to_retire, timeout=10):
         proposal = self.get_any_active_member().propose(remote_node, proposal_body)
         self.vote_using_majority(remote_node, proposal, careful_vote)
 
-    def trust_node(
-        self, remote_node, node_id, valid_from, validity_period_days=None, timeout=3
+    def trust_nodes(
+        self,
+        remote_node,
+        node_ids,
+        valid_from,
+        validity_period_days=None,
+        wait_for_commit=True,
+        timeout=3,
     ):
-        proposal_body, careful_vote = self.make_proposal(
-            "transition_node_to_trusted",
-            node_id=node_id,
-            valid_from=valid_from,
-            validity_period_days=validity_period_days,
-        )
+        proposal_body = {"actions": []}
```

Comment:
This proposal can trust multiple node at once, whereas the infra could only trust one single node per proposal previously.